### PR TITLE
loads on every mouseenter

### LINKED ISSUE #18: Disregard GitHub's UI status and try loading on every mouseenter
I've found that the icon may not be up to date. For example when the computer goes to sleep without notifications and it wakes up later; the icon won't have had time to update yet and NPG won't open. So maybe it makes sense to **run `openPopup` whether `.unread` is there or not.**

### PR DESCRIPTION
**Pull request by tanmayrajani**
Fixes #18

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,6 +68,13 @@ async function openPopup() {
...
 		if (notificationsList.length === 0) {
 			return;
+		} else if (!select('.notification-indicator .mail-status').classList.contains('unread')) {
+			const newDom = await fetch(document.URL, {
+				credentials: 'include'
+			}).then(r => r.text()).then(domify);
+
+			indicator.parentNode.replaceChild(select('.notification-indicator', newDom), indicator);
+			init();
 		}
```
Comment (tanmayrajani) [inline index.js]: for #18, I tried bringing back the unread indicator this way. 

@bfred-it I'm not sure this is a good way.. any better way to do this?

Comment (bfred-it) [reply index.js]: I'll push a commit to this branch in a few minutes

File: index.js [hunk 2]
```diff
@@ -69,6 +68,13 @@ async function openPopup() {
 		notificationsList = select.all('.boxed-group', notificationsPage);
 		if (notificationsList.length === 0) {
 			return;
+		} else if (!select('.notification-indicator .mail-status').classList.contains('unread')) {
+			const newDom = await fetch(document.URL, {
+				credentials: 'include'
+			}).then(r => r.text()).then(domify);
+
+			indicator.parentNode.replaceChild(select('.notification-indicator', newDom), indicator);
+			init();
```
Comment (bfred-it) [inline index.js]: Why is this needed?

Comment (tanmayrajani) [reply index.js]: I think when I replace the `.notification-indicator`, the old event listeners are gone (at least that's what I interpreted when it didn't work the next time). Hence, before I hover on the icon next time, it should register the events to keep it funcitoning..

**Comment by bfred-it** [comment]
How about this?

File: index.js [hunk 3]
```diff
@@ -37,6 +37,15 @@ function isOpen() {
 /**
  * Extension
  */
+function restoreUnreadIndicator() {
+	const indicator = select('.notification-indicator');
+	const status = select('.mail-status', indicator);
+	if (!status.classList.contains('.unread')) {
```
Comment (tanmayrajani) [inline index.js]: would be `contains('unread')` (and not `.unread`), right? :smiley:

**Comment by tanmayrajani** [comment]
yes, this is better! we should merge this!

File: index.js (hunk 3 above)
Comment (bfred-it) [reply index.js]: Ah yeah, I had initially written `status.matchs('.unread')`
